docs/HISTORY.txt

```diff
@@ -5,7 +5,8 @@ Changelog
 2.14.3 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Fix transfer of date and time objects [Nachtalb]
+- Fix timezone transfer of datetime and time objects [Nachtalb]
```

Comment:
Is the duplicate entry on purpose?

Reply:
No, it's 2 different things. On one hand, we fix how timezones are transferred and on the other hand, we converted times and dates to datetimes instead of keeping it as a time and date object.

Reply:
Thanks for the clarification.